app/views/hyrax/base/_form_media.html.erb

```diff
@@ -1,4 +1,8 @@
 <% if f.object.persisted? && f.object.member_ids.present? %>
   <%= render 'form_representative', f: f %>
   <%= render 'form_thumbnail', f: f %>
+  <%# display the rendering dropdown only if the representitive responds to image? (ie. Universal Viewer is used) %>
+  <% if !curation_concern.representative.blank? && curation_concern.representative.image? %>
```

Comment:
Prefer `#present?` over the negation of `#blank?`

Reply:
Done :)